common.sh

```diff
@@ -187,9 +187,29 @@ create_instance()
 get_instance_ip()
 {
     execution_seq=$((${execution_seq}+1))
-    INSTANCE_IPS=$(aws ec2 describe-instances --instance-ids ${INSTANCE_IDS[@]} \
-                        --query "Reservations[*].Instances[*].PrivateIpAddress" \
-                        --output=text)
+    local retry=20
+    local ret=0
+    set +e
+    while [ $retry -ge 0 ]; do
+        INSTANCE_IPS=$(aws ec2 describe-instances --instance-ids ${INSTANCE_IDS[@]} \
+                            --query "Reservations[*].Instances[*].PrivateIpAddress" \
+                            --output=text)
+        ret=$?
+        if [ "$ret" -eq 0 ] && [ -n "$INSTANCE_IPS" ]; then
+            echo "get instance ips successfully."
```

Comment:
This print doesn't seem useful.  If we want debugging here, let's at last print out the IPs?

Reply:
Printed the IPs in the latest revision.